README.md

```diff
@@ -1,3 +1,147 @@
-# Opsgenie App
+# Mattermost/OpsGenie Integration
 
-A Opsgenie app for Mattermost.
+* [Feature summary](#feature-summary)
+* [Set up](#set-up)
+    * [Installation HTTP](#installation-http)
+    * [Installation Mattermost Cloud](#installation-mattermost-cloud)
+    * [Configuration](#configuration)
+* [Admin guide](#admin-guide)
+    * [Slash commands](#slash-commands)
+* [End user guide](#end-user-guide)
+    * [Get started](#get-started)
+    * [Use /opsgenie commands](#use-genie-commands)
+* [Development environment](#development-environment)
+    * [Manual installation](#manual-installation)
+    * [Install dependencies](#install-dependencies)
+    * [Run the local development environment](#run-the-local-development-environment)
+  * [Run the local development environment with Docker](#run-the-local-development-environment-with-docker)
+
+This application allows you to integrate OpsGenie to your Mattermost instance, letting you know when a new alert is created, as well as getting notified about alert updates. Also, allows the user to create new alerts, add notes to alerts, close alerts, assign alerts, etc. without moving from the Mattermost window.
+
+# Feature summary
+
+**OpsGenie to Mattermost notifications:** Link your Mattermost channels with the OpsGenie Teams you want to see, so you and your team can get notifications about the creation and updates of each alert.
+
+# Set up
+
+## Installation HTTP
+
+To install, as a Mattermost system admin user, run the command ``/apps install http OPSGENIE_API`` in any channel. The ``/genie`` command should be available after the configuration has been successfully installed.
+
+The ``OPSGENIE_API`` should be replaced with the URL where the OpsGenie API instance is running. Example: ``/apps install http https://myapp.com/manifest.json``
+
+## Installation Mattermost Cloud
+
+To install, as a Mattermost system admin user, run the command ``/apps install listed genie`` in any channel. The ``/genie`` command should be available after the configuration has been successfully installed.
```

Comment:
```suggestion
To install, as a Mattermost system admin user, open the Mattermost marketplace and choose "Install" next to the OpsGenie app. The ``/genie`` command should be available after the configuration has been successfully installed.
```

Reply:
Is the ``/genie`` command available after the **configuration** has been installed or the **integration** has been installed? This step appears to be installing the integration rather than configuring it.

Reply:
It's available after the app/integration is installed. The sysadmin then uses `/genie configure` to configure the app